tests/test_validate.py

```diff
@@ -4,17 +4,21 @@
 import authorize
 import validate
 import config
+from datetime import datetime, timedelta
 
 import pytest
 import json
+import jwt
 
 @pytest.mark.run(order=3)
 def test_validate_token():
     user_email = "creep@radiohead.ed"
     passwd = "love"
     usr_dict = {'email': user_email, 'password': passwd}
     auth = authorize.authorize(usr_dict, None)
-    token = auth['body']['auth']['token']
+    token = auth['body']['token']
+
+    print("token is " + token)
```

Comment:
Debug leftover?

Reply:
Yeah, that looks like a debug leftover. We should probably actually add more print statements to our tests in general in the future so when something goes wrong we have a sort of log of what happened but that is mostly an unrelated goal.